domain-ee/ee-ep-merge-app/src/python_src/util/contentions_util.py

```diff
@@ -7,9 +8,25 @@ def __init__(self, message=None):
         super().__init__(self.message)
 
 
+class CompareException(BaseException):
+    def __init__(self, message=None):
+        self.message = message
+        super().__init__(self.message)
+
+
 class ContentionsUtil:
     @staticmethod
     def merge_claims(pending_contentions: Response = None, ep400_contentions: Response = None):
         if pending_contentions and ep400_contentions:
-            return [*pending_contentions.contentions, *ep400_contentions.contentions]
+            try:
+                return [*pending_contentions.contentions, *ContentionsUtil.new_contentions(pending_contentions.contentions, ep400_contentions.contentions)]
+            except CompareException:
+                raise
```

Comment:
Is there any reason to catch this `CompareException` and re-raise the original? It seems easier to just leave out the try, except, and raise lines.